### PR TITLE
Adding week 6 blog post

### PR DESCRIPTION
**Pull request by xtanion**
The blog is visible in `blogs.html`

### CODE REVIEW

**Review by m-agour — COMMENTED** [review]
Hello @xtanion,
Good post without typos or any issues.

**Review by ganimtron-10 — COMMENTED** [review]
Hello @xtanion ,
Good Blog! Everything LGTM.
Thanks!

File: docs/source/posts/2022/2022-07-25-week-6-shivam.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
+Week 6 - Extracting the animation data
+======================================
+.. post:: July 25 2022
```
Comment (skoudoro) [inline docs/source/posts/2022/2022-07-25-week-6-shivam.rst:3]: missing empty line above

File: docs/source/posts/2022/2022-07-25-week-6-shivam.rst [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+
+- This week, it was all about reading docs and extracting the animation data from buffers.
+
+- Currently, the glTF class can extract simple node transformation and morphing data, As they are stored in the ``Animations`` of the glTF file.
```
Comment (skoudoro) [inline docs/source/posts/2022/2022-07-25-week-6-shivam.rst:14]: > As they are stored in the

no need of uppercase

File: docs/source/posts/2022/2022-07-25-week-6-shivam.rst [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+
+What is coming up next week?
+----------------------------
+
+As of now, we've decided the following:
+
+- Create a custom Interpolator.
+
+Other tasks are yet to be decided.
```
Comment (skoudoro) [inline docs/source/posts/2022/2022-07-25-week-6-shivam.rst:33]: please, remove this, and propose tasks that you know you will do. 

This sentence makes people believe that you are done and just waiting without any task idea. Same remark for @m-agour and @ganimtron-10. Please, avoid this sentence all and explain what you REALLY plan to do. Our meetings are just to update the plan.

File: docs/source/posts/2022/2022-07-25-week-6-shivam.rst [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+What did you do this week?
+--------------------------
+
+- This week, it was all about reading docs and extracting the animation data from buffers.
+
+- Currently, the glTF class can extract simple node transformation and morphing data, As they are stored in the ``Animations`` of the glTF file.
+
+- Skinning (Skeletal Animation) data is stored in ``Nodes`` inside the skin parameter. We shall be able to load that before our next meeting on Wednesday.
+
+- Created a `tutorial <https://github.com/xtanion/fury/blob/gltf-anim-merge-kf/docs/tutorials/01_introductory/viz_simple_gltf_animation.py>`_ using keyframe animations (`#626`_.) and adding multiple ``timelines`` into a main ``timeline`` as suggested by Mohamed.
```
[the remark this answers is not in the document]
Comment (skoudoro) [reply docs/source/posts/2022/2022-07-25-week-6-shivam.rst:18]: still here

**Review by skoudoro — COMMENTED** [review]
Hi @xtanion,

See below for some comments. Thanks for the future update.

**Comment by skoudoro** [comment]
Thank you for the update. Merging

**Pull request merged**